libs/langchain-community/src/vectorstores/azure_cosmosdb.ts

```diff
@@ -205,16 +233,24 @@ export class AzureCosmosDBVectorStore extends VectorStore {
 
   /**
    * Removes specified documents from the AzureCosmosDBVectorStore.
-   * @param ids IDs of the documents to be removed. If no IDs are specified,
-   *     all documents will be removed.
+   * @param filterOrIds A MongoDB filter object or list of IDs for the
+   *     documents to be removed. If no IDs or filter are specified, all
+   *     documents will be removed.
    * @returns A promise that resolves when the documents have been removed.
    */
-  async delete(ids?: string[]): Promise<void> {
+  async delete(
+    filterOrIds?: string[] | Filter<MongoDBDocument>
```

Comment:
Would prefer if this was either an array of `ids` or an `options: { filter: Filter<MongoDBDocument> }` param

Reply:
Done!